vendor/engines/secure_rooms/app/controllers/secure_rooms/facility_occupancies_controller.rb

```diff
@@ -0,0 +1,22 @@
+module SecureRooms
+
+  class FacilityOccupanciesController < ApplicationController
+
+    include TabCountHelper
```

Comment:
ah, not in this PR, that bit in tabnav_occupancy is where it will be needed immediately following